ajax/libs/docsify/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "docsify",
+  "filename": "docsify.min.js",
+  "version": "1.1.5",
+  "description": "A magical documentation generator.",
+  "homepage": "https://docsify.js.org",
+  "author": "qingwei-li <cinwell.li@gmail.com> (https://github.com/QingWei-Li)",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/QingWei-Li/docsify.git"
+  },
+  "keywords": [
+    "doc",
+    "docs",
+    "documentation",
+    "creator",
+    "generator"
+  ],
+  "npmName": "docsify",
+  "npmFileMap": [
+    {
+      "basePath": "lib",
+      "files": [
+        "docsify.min.js",
```

Comment:
I think this line can be modified to `docsify*`, because there is another file named `docsify.js`.
And you can also ask the author for which files should be added.